app/helpers/blacklight_helper.rb

```diff
@@ -75,12 +74,6 @@ def series_title_results(solr_parameters)
                            'electronic_access_1display,cataloged_tdt,series_display'
   end
 
-  def only_home_facets(solr_parameters)
-    return if search_parameters?(solr_parameters)
-    solr_parameters['facet.field'] = home_facets
-    solr_parameters['facet.pivot'] = []
-  end
-
   # only fetch facets when an html page is requested
   def html_facets(solr_parameters)
     return if blacklight_params[:format].nil? || blacklight_params[:format] == 'html' ||
```

Comment:
what is the blacklight_params[:format] == 'html' ?

Reply:
that code doesn't get changed here, but it's the format of the page requested ie:
https://catalog.princeton.edu/catalog?format=html&search_field=all_fields&q=my+favorite+book
https://catalog.princeton.edu/catalog?format=json&search_field=all_fields&q=my+favorite+book

Reply:
thanks